_harp/academy/_apply.md

```diff
@@ -2,33 +2,26 @@
 
 The Spring 2017 programme starts on Monday 20th February. 
 
-Applications will open in October.
+Applications will open on November 1st and close on November 30th.
 
 ## The application process
 
 + **[Expression of interest](interest.html)** - takes two minutes to complete.
-+ Application - opening soon
++ Application - opening on November 1st.
 
-We may also request a short in-person paired programming session (remote sessions can also be arranged).
+We will expect you to have started on your journey to become a software developer before you join us on the programme.
 
-We offer places throughout the months leading up to the application deadline. Early application is recommended, so we know that you are interested and can keep an eye out for you in our community.
+There are a number of ways to demonstrate your readiness for the programme:
 
-## Minimum application requirements
++ Completing and creating JavaScript kata on [Codewars](http://www.codewars.com/?language=javascript)
++ Making progress with [Free Code Camp](http://www.freecodecamp.com/) *
++ Creating your own website using [GitHub pages](https://pages.github.com/) *
++ Participation in one of our [Meetup groups](http://www.meetup.com/founderscoders/)
 
-We will expect you to have started on your journey to become a software developer before you join us on the programme. Our minimum requirements provide the guidance you need to achieve this:
+\* While learning CSS and JavaScript, we recommend against using frameworks like Bootstrap and jQuery.
 
-+ Reach at least 5 kyu on [Codewars](http://www.codewars.com/?language=javascript);
-+ Create some kata on [CodeWars](http://www.codewars.com/kata/new);
-+ Contribute to [our community on GitHub](https://github.com/codingforeveryone)
-(e.g. our [READMEs](https://github.com/codingforeveryone/READMEs));
-+ Remain active in [our Gitter chatrooms](https://gitter.im/codingforeveryone)
+We have no minimum criteria for application, but you are unlikely to secure a place on the programme without a considerable amount of prior effort. 1-200 hours of self-study is probably typical.
```

Comment:
I reckon 100-200 is a little clearer.

Also.... If we're still going to dismiss people out of hand who haven't reached 5 kyu on codewars for example, this might be unhelpful as people won't know where to concentrate their efforts - they might be amazing in codecademy and read all of eloquent JS but not touched codewars... 😟